lib/tire/index.rb

```diff
@@ -339,7 +339,7 @@ def remove(*args)
       options.select! { |k, | [:routing, :parent, :replication, :consistency, :refresh, :timeout].include? k }
       params_encoded = options.empty? ? '' : "?#{options.to_param}"
 
-      url    = "#{self.url}/#{type}/#{Utils.escape(id)}#{params_encoded}"
+      url    = "#{self.url}/#{Utils.escape(type)}/#{Utils.escape(id)}#{params_encoded}"
```

Comment:
Is there any issue with escaping it twice? (because of line 239?)

Reply:
239 is a blank line

Reply:
329.

You're right now we're double escaping.